src/index.ts

```diff
@@ -1,5 +1,13 @@
 const assert = require('assert');
-const { types } = require('util');
+let { types } = require('util');
+if (!types) {
+  // Node.js < 10
+  types = {
+    isExternal(x: any): boolean {
+      return Object.prototype.toString.call(x) === '[object Object]';
+    }
+  }
+}
```

Comment:
Hmm, very slightly gross as a polyfill, I might prefer something like:

```ts
const types = require('util').types || {
  isExternal(x: any) ...
};
```

I would merge anyway, but the build output is going to conflict, so take it or leave it when you rebase and rebuild the output.

Reply:
You’re right, that’s more elegant. Changed.